aspnetcore/razor-pages/ui-class.md

```diff
@@ -221,3 +221,30 @@ Suppose *RazorUIClassLib/Pages/Shared* contains two partial files: *_Header.csht
   <partial name="_Footer">
 </body>
 ```
+
+## Create a Razor class library with static assets
+
+Razor class libraries (RCL) frequently require companion static assets that can be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available to a consuming app.
+
+To include companion assets as part of a Razor class library, create a *wwwroot* folder in the class library and include any required files in that folder.
+
+When packing a Razor class library, all companion assets in the *wwwroot* folder are included in the package automatically and are made available to apps referencing the package.
+
+### Consume content from a referenced Razor class library
+
+The files included in the *wwwroot* folder of the Razor class library are exposed to the consuming app under the prefix `_content/{LIBRARY NAME}/`. `{LIBRARY NAME}` is the library project name converted to lowercase with periods (`.`) removed. For example, a library named *Razor.Class.Lib* results in a path to static content at `_content/razorclasslib`.
+
+The consuming app references static assets provided by the library with `<script>`, `<style>`, `<img>`, and other HTML tags. The consuming app must have static file support enabled. For more information, see <xref:fundamentals/static-files>.
+
+### Multi-project development flow
+
+When the app runs:
+
+* The assets stay in their original folders.
+* Any change within the class library *wwwroot* folder is reflected in the app without rebuilding.
```

Comment:
"Without rebuilding" is ambiguous, could refer to the app or the class library.  Here I am guessing the reference is to building the app, but in the next paragraph it sounds like the reference is to building the library.